src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -29,6 +29,20 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
     MTRDeviceStateUnreachable = 2
 };
 
+/**
+ * This bitmask is used to specify the type of logs requested from this device.
+ *
+ * The logs types are : End User Support, Network Diag and Crash logs.
```

Comment:
```suggestion
 * The logs types are : End User Support, Network Diagnostics and Crash logs.
```
Please don't use inscrutable abbreviations if we can avoid it.